src/utils/configurations/configurationsContext.js

```diff
@@ -0,0 +1,253 @@
+import { useAlert, useDataEngine, useDataQuery } from '@dhis2/app-runtime'
+import {
+    Box,
+    CenteredContent,
+    CircularLoader,
+    Layer,
+    NoticeBox,
+} from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React, { useState, useContext } from 'react'
+import { configurationsReducer } from './configurationsReducer.js'
+
+// todo:
+// 4. set up a datastore key if one isn't already with default config (RWDQA-50)
+
+const DATASTORE_KEY =
+    process.env.REACT_APP_DHIS2_APP_DATASTORE_KEY || 'configurations'
```

Comment:
nice. not sure if we want to add a note about this to the README, or if we want to plan to take this away after initial development?

Reply:
I'll add this to the README, then we can reevaluate after development 🙂 I think it should be fine if we happen to forget, but maybe I'll add a console log to remind someone if they're using a nonstandard key